assets/markdown/remapURLs.js

```diff
@@ -0,0 +1,6 @@
+module.exports = function (value) {
+  var userRegex = /(<a href=")(\/[u]\/[a-zA-Z_\/-]+)(">)(.*)(<\/a>)/g;
+  var usersReplaced = value.replace(userRegex, "$1https://www.reddit.com$2$3$4$5 ($1$2$3FlairHQ$5)");
+  var subRegex = /(<a href=")(\/[r]\/[a-zA-Z_\/-]+)(">)(.*)(<\/a>)/g;
```

Comment:
I should probably just change it to essentially be a username again, so it doesn't allow spaces etc. That will solve the other issue as well. Will look later.